coordinator/src/services/translation/request/dispense/dispense-notification.ts

```diff
@@ -277,6 +295,15 @@ export function getOrganisationPerformer(fhirFirstMedicationDispense: fhir.Medic
   return fhirFirstMedicationDispense.performer.find(p => p.actor.type === "Organization")
 }
 
+export function getMedicationDispenseContained(
+  fhirFirstMedicationDispense: fhir.MedicationDispense
+): fhir.MedicationRequest | undefined {
+  if (fhirFirstMedicationDispense.contained.length === 1) {
```

Comment:
with validation infront of this, we should be safe to assume theres always a medicationRequest inside the medicationDispense at this point

Reply:
done